tests/concurrent/concurrent.go

```diff
@@ -256,7 +256,7 @@ func main() {
 			Name:       name,
 			VMFullName: name,
 
-			ParentImageID: parent,
+			// ParentImageID: parent,
```

Comment:
What's the purpose of this change?

Reply:
`ParentImageID` has been deleted as a field - it was being used to pass the parent ID through to the hacky code that was constructing the image disk reference. Now that that is done in a structured manner the field is no longer needed.